marahgh/assignment3/Grid.cpp

```diff
@@ -0,0 +1,71 @@
+#include "Grid.h"

```

Comment:
actually this is a problem that i have since in the university i learn at teach us to implement almost everything we use so it didn't come to my mind to use it, this is a problem i'm getting better in conquering it because of these assignments :P thank you! :D

Reply:
Again, I'm not saying it's wrong, there are arguments both ways. Just want you to think what are the advantages are of one vs the other.
No problem, happy to help :)